app/notification/routes.py

```diff
@@ -18,7 +20,7 @@
 )
 
 
-@notification_bp.route("/", methods=["POST", "GET"])
+@notification_bp.route("", methods=["POST", "GET"])
```

Comment:
Why does this accept GET when it is expecting a POST?

Reply:
For some reason, it wasn't working without the GET method but now I have tried without it, it works so updated!